srcpkgs/nnn/template

```diff
@@ -14,6 +14,84 @@ changelog="https://raw.githubusercontent.com/jarun/nnn/master/CHANGELOG"
 distfiles="https://github.com/jarun/nnn/archive/v${version}.tar.gz"
 checksum=15acaf9a88cfb5a2a640d3ef55a48af644fba92b46aac0768efe94c4addf7e3f
 
+# Package build options
+build_options="readline \
```

Comment:
the style for multi-line strings is just:
```sh
var_name="line one here
 line two here
 line three here"
```
(note the space starting each line and the lack of backlashes)

you also don't need to put each one on its own line

Reply:
Oh thanks! I'll update the PR when I am home again. 😄